next.config.js

```diff
@@ -1,4 +1,8 @@
 /** @type {import('next').NextConfig} */
-const nextConfig = {}
+const nextConfig = {
```

Comment:
This as well as the changes to the prefetch queries were all attempts to avoid the weird caching behaviour. This specifically prevents next from caching at all in a dev env. I have logged a ticket to investigate this further so we do not have random things for the same purpse everywhere in #62